app/views/migration/_form.html.erb

```diff
@@ -39,7 +39,7 @@
     <%= field_set_tag(i18n.t('legends.all_objects')) do %>
       <%= field_set_tag(i18n.t('legends.each_object', number: 1)) do %>
         <%= agg.simple_fields_for(:edm_isShownBy) do |wr| %>
-          <%= wr.input(:media, label: false) %>
+          <%= wr.input(:media, label: false, input_html: { accept: "#{EDM::WebResource.allowed_extensions}, #{EDM::WebResource.allowed_mime_types}" }) %>
```

Comment:
Create a helper method responsible for generating this string for the `accept` attribute, and make it responsible for the `join(', ')`, not the methods in `EDM::WebResource`.